### PR TITLE
Update Portuguese translator

### PR DESCRIPTION
**Pull request by MestreWalla**
Better context, add some missing translation

### CODE REVIEW

**Review by prem-k-r — APPROVED** [review]

File: languages.js [hunk 1]
```diff
@@ -1594,7 +1620,7 @@ function applyLanguage(lang) {
         { id: 'perplexity', key: 'perplexity' },
         { id: 'firefly', key: 'firefly' },
         { id: 'github', key: 'github' },
-        { id: 'googleAppsHover', key: 'googleAppsHover' },
+        { id: 'googleAppsHover', key: 'googleAppsHover' }
```
[the remark this answers is not in the document]
Comment (itz-rj-here) [reply languages.js:1623]: Why you removed a comma from the code?

**Review by itz-rj-here — APPROVED** [review]
Ohh It's ok np.

File: languages.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (XengShi) [reply languages.js:1623]: If this is last element commas are not needed

[the remark this answers is not in the document]
Comment (prem-k-r) [reply languages.js:1623]: It's fine, it's not really needed, doesn't affect anything

[the remark this answers is not in the document]
Comment (itz-rj-here) [reply languages.js:1623]: Yeah I was reviewing the code from my phone, thats way i didn't saw the last line sorry 😅

**Pull request merged**